seeds/balancer/balancer_transfers_bpt_seed.csv

```diff
@@ -0,0 +1,34 @@
+ethereum,0x9f2b223da9f3911698c9b90ecdf3ffee37dd54a8,0x806977d6241ec7043986dc6aeda3d9c9aa57c2f63a01c5693f5ffbf7cc511bb5,352,2021-07-04T04:45:40Z,2021-07-04T00:00:00Z,12759000,0x485dcba4594f54ae23478bb32aa411012e33d62a,0x45729f8333e0ad91811a10de1f0d1b5b72ed3de8,1.1929882732571E+36
+optimism,0x6222ae1d2a9f6894da50aa25cb7b303497f9bebd,0xc3107f9c6b91dfa30d9aef8d350756ae960efa307cc5afc8e4909cc7899eba9d,128,2022-12-14T23:16:30Z,2022-12-14T00:00:00Z,49712483,0x4854c82b12a389051a4bb55ab24b1da3ce8a2b5b,0xdc785bc8280d8fdb89aeb4980e061e34a34e71d4,6.8843671950394E+31
+optimism,0x6222ae1d2a9f6894da50aa25cb7b303497f9bebd,0x98ca6154da302de973916224286bd0ad639f07d0c7bf4222a2cb3de79fb3e997,19,2022-12-14T22:25:18Z,2022-12-14T00:00:00Z,49701598,0x0000000000000000000000000000000000000000,0xce88686553686da562ce7cea497ce749da109f9f,3.1077393919726E+27
+optimism,0x4fd63966879300cafafbb35d157dc5229278ed23,0x5f33ae2f08672fac5df4c6a3416922a4ee7f1d0b973d5628de457d10d7d29213,0,2022-12-14T22:58:38Z,2022-12-14T00:00:00Z,49709176,0x4f811cdd070b05fe1861d34ed001b346fdfddc2d,0x0000000000000000000000000000000000000000,5.3722477455098E+30
+optimism,0x4fd63966879300cafafbb35d157dc5229278ed23,0xab99372ba82908c3ae394c8b95096d5edde726ff1f867ef35f48f03d06b25ed2,0,2022-12-14T23:37:34Z,2022-12-14T00:00:00Z,49717267,0xb4ffa1a4b589e2b19ec47b7d5716e1076d791a57,0x0000000000000000000000000000000000000000,4.9286681093321E+30
+optimism,0x4fd63966879300cafafbb35d157dc5229278ed23,0x09c68a2d288209c26bfbe27cd93716f34a4b28331b2d6d9815b769433c68a770,2,2022-12-14T22:22:03Z,2022-12-14T00:00:00Z,49700712,0x38f79beffc211c6c439b0a3d10a0a673ee63afb4,0xcedd02d15d9e003fad7184c27adcc1cec2339240,4.9385214722120E+30
+polygon,0x4e7f40cd37cee710f5e87ad72959d30ef8a01a5d,0x6c379ae3071b4e3f7dd505d05712c16aa11b3701e557d5a251525debac025ad3,164,2021-11-29T03:42:52Z,2021-11-29T00:00:00Z,21923092,0x0000000000000000000000000000000000000000,0xb4e92d96bb76101c2e6d6dfa73ae6553cd2d9eff,1.0783318276699E+31
+polygon,0xcf354603a9aebd2ff9f33e1b04246d8ea204ae95,0xad968f2f40c333cb2f60e44da9c787a693fcde22929a3ce57f63aa719b3b138e,111,2021-11-29T23:23:37Z,2021-11-29T00:00:00Z,21952772,0xf96a7dd64e324c5ecfd03f1eed856d403d1b4b77,0x0000000000000000000000000000000000000000,3.1399879399790E+32
+polygon,0xba07d63875d0c2319d43cbb8a897f89bdafd7e87,0xb5d4edf86e429600666fe80f2ae2d5e722754a7b7e43a6c9bc9df6b920178ee3,135,2021-11-29T08:27:14Z,2021-11-29T00:00:00Z,21930050,0x0000000000000000000000000000000000000000,0xce6a15c98ac6603d4148f5267899b61b810c819d,4.7100193941702E+31
+polygon,0x0d34e5dd4d8f043557145598e4e2dc286b35fd4f,0x8f19070ad3590a1dab8f9f0827fe10ee8151cc94610d95885a53748874407bd1,0,2021-11-29T13:06:14Z,2021-11-29T00:00:00Z,21937400,0x2b626bb1e593ed9a49e2277c7b7095099058ee71,0x0000000000000000000000000000000000000000,3.9280475960983E+35
```

Comment:
Here is the last unmatched seed.
https://dune.com/queries/1853053?d=1


test_description | result (model) | expected (seed) | evt_tx_hash | evt_index | block_date
-- | -- | -- | -- | -- | --
equality test: evt_block_number | 21937404 | 21937400 | 0x8f19070ad3590a1dab8f9f0827fe10ee8151cc94610d95885a53748874407bd1 | 0 | 2021-11-29 00:00

@Stefenon You can have a check, and after this fixes, I think the PR is at ready-for-merging stage.

Reply:
According to [Polygonscan](https://polygonscan.com/tx/0x8f19070ad3590a1dab8f9f0827fe10ee8151cc94610d95885a53748874407bd1) the block number 21937404 is indeed the correct. @Stefenon can you fix the seed file?

Reply:
Thank you for verifying it @Hosuke @mendesfabio, I added a commit changing it.